scripts/create-nodejs-16.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
+readonly PROG_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+readonly ROOT_DIR="$(cd "${PROG_DIR}/.." && pwd)"
+readonly STACK_DIR="${ROOT_DIR}/stack-nodejs-16"
```

Comment:
I think we could cut down on the duplication of the stack creation scripts and just have one if we make the `STACK_DIR` configurable via a flag, with the default being `${ROOT_DIR}/stack` if nothing is provided. 


This way, we can then amend the [`create-stack` action](https://github.com/paketo-buildpacks/github-config/blob/main/actions/stack/create-stack/action.yml) to take in an optional stack directory. This action gets used in the [release creation workflow](https://github.com/paketo-buildpacks/jammy-full-stack/blob/main/.github/workflows/create-release.yml) (that we will add eventually)